ames/matchers/caltechdata.py

```diff
@@ -11,6 +11,123 @@
 import requests
 
 
+def edit_subject(record, token, correction_subjects, test=True):
+    if test:
+        rurl = "https://data.caltechlibrary.dev/api/records/" + record
+    else:
+        rurl = "https://data.caltechlibrary.dev/api/records/" + record
+
+    headers = {
+        "Authorization": "Bearer %s" % token,
+        "Content-type": "application/json",
+    }
+
+    data = requests.get(rurl, headers=headers).json()
+
+    json_string = json.dumps(data["metadata"], indent=4)
+
+    metadata = get_metadata(
+        record,
+        production=False,
```

Comment:
You're going to need to set this based on the `test` variable (or change it to `production for consistency)